src/calculateRentalCost.js

```diff
@@ -27,8 +27,25 @@
  * @return {number}
  */
 
+calculateRentalCost(7);
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40;
+  let price = 0;
+
+  for (let i = 0; i < days; i++) {
+    price = price + rentalCost;
+  }
```

Comment:
```suggestion
  const rentalCost = 40 * days;
```
Do you really need this loop? 
Can't you just multiply days with 1 day price?
By the way, would be great to create a variable for `40` to make your code cleaner